projects/collection-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -195,13 +198,22 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
     this.helperService.initialize(channel);
   }
 
-  getFrameworkDetails(categoryDefinitionData) {
+  async getFrameworkDetails(categoryDefinitionData) {
```

Comment:
Why you have added `async` this function is calling multiple place in this will it not effect?

Reply:
async and await is removed we will find a alternative way